packages/cli/cli.js

```diff
@@ -12,22 +12,34 @@ import helpMe from 'help-me'
 
 import { logo } from './lib/ascii.js'
 
-const program = commist()
+const program = commist({ maxDistance: 1 })
```

Comment:
Why this change?

Reply:
I would change it to 2. maxDistance 4 is too much.
`platformatic db fadk` shouldn't be parsed as a `platformatic db start`

Reply:
Exactly for this reason. With the change from this PR, making something like `mgratm` for `migratinon` sounds a little too much.

Let's change it to `2`, as this is the behavior of `git.autocorrect` setting as well, which people may be already familiar with - https://github.com/git/git/blob/c48035d29b4e524aed3a32f0403676f0d9128863/help.c#L657